app/assets/javascripts/account/AccountService.js.coffee

```diff
@@ -92,6 +93,12 @@ AccountService = ($state, $auth, $modal, $http, ShortFormApplicationService) ->
       return
     )
 
+  Service.getMyApplications = ->
```

Comment:
hmm.. I see your point. I just always prefer 3rd person naming as opposed to 1st person -- that way we know which player it is. Right now we have one player which is the user, but what if in the future we create an agent portal or a admin portal or something, in which case the 3rd person naming might be helpful. 

But if we ever add more players, we can easily rename things as they are needed. So I'm cool with it staying this way, I guess so long as all exclusively logged-in user functions follow the same naming.